### PR TITLE
Remove code referencing Vertica and delete deprecated jobs.

### PR DESCRIPTION
**Pull request by bmedx**
Analytics Pipeline Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] If you have a migration please contact data engineering team before merging.
  - [ ] Before merging run full acceptance tests suite and provide URL for the acceptance tests run.
  - [ ] A member of data engineering team has approved the pull request.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -56,36 +55,7 @@ edx.analytics.tasks =
     student_engagement = edx.analytics.tasks.data_api.student_engagement:StudentEngagementTask
 
     # warehouse:
-    event-type-dist = edx.analytics.tasks.warehouse.event_type_dist:PushToVerticaEventTypeDistributionTask
-    load-course-catalog = edx.analytics.tasks.warehouse.load_internal_reporting_course_catalog:PullDiscoveryCoursesAPIData
```
Comment (brianhw) [inline setup.cfg:60]: I think you will still need this.   Or something in this file, so its tasks get loaded.

File: edx/analytics/tasks/warehouse/load_internal_reporting_course_catalog.py [hunk 2]
```diff
@@ -499,28 +498,6 @@ def requires(self):
...
-    @property
-    def table(self):
-        return 'd_program_course'
-
-    @property
-    def columns(self):
-        return ProgramCourseRecord.get_sql_schema()
-
-
 class CourseSeatRecord(Record):
```
Comment (brianhw) [inline edx/analytics/tasks/warehouse/load_internal_reporting_course_catalog.py:501]: So I'm guessing you made a conservative cut, but I expect that there is a subset of such Records and the related tasks to keep to support enrollment.  And that this is likely one of the ones that could be deleted.  The include in enrollment mentions 
```
from edx.analytics.tasks.warehouse.load_internal_reporting_course_catalog import (
    CoursePartitionTask, LoadInternalReportingCourseCatalogMixin, ProgramCoursePartitionTask
)
```
So we should only need the mixin, and the stuff related to the two tasks.   Should...    

Admitted each of the two has some upstream stuff in the file to keep.  

Anyway, we can clean this out later, if you prefer to do it in stages.

**Review by brianhw — COMMENTED** [review]
Looks good -- only one blocker.

File: setup.cfg (hunk 1 above)
Comment (bmedx) [reply setup.cfg:60]: Good catch, added that back

File: edx/analytics/tasks/warehouse/load_internal_reporting_course_catalog.py (hunk 2 above)
Comment (bmedx) [reply edx/analytics/tasks/warehouse/load_internal_reporting_course_catalog.py:501]: I think I'll leave that for later and try to get this through acceptance tests as-is. If I still have time after that I'll take a swipe at it and try to whittle off a little more.

File: setup.cfg [hunk 3]
```diff
@@ -56,36 +55,8 @@ edx.analytics.tasks =
...
-    load-d-user-course = edx.analytics.tasks.warehouse.load_internal_reporting_user_course:LoadUserCourseSummary
-    load-events = edx.analytics.tasks.warehouse.load_internal_reporting_events:TrackingEventRecordDataTask
-    load-f-user-activity = edx.analytics.tasks.warehouse.load_internal_reporting_user_activity:LoadInternalReportingUserActivityToWarehouse
-    load-ga-permissions = edx.analytics.tasks.warehouse.load_ga_permissions:LoadGoogleAnalyticsPermissionsWorkflow
-    load-google-sheet-vertica = edx.analytics.tasks.warehouse.load_google_sheet_to_warehouse:LoadGoogleSpreadsheetsToVerticaWorkflow
     load-google-sheet-snowflake = edx.analytics.tasks.warehouse.load_google_sheet_to_warehouse:LoadGoogleSpreadsheetsToSnowflakeWorkflow
-    load-insights= edx.analytics.tasks.warehouse.load_warehouse_insights:LoadInsightsTableToVertica
-    load-internal-active-users = edx.analytics.tasks.warehouse.load_internal_reporting_active_users:LoadInternalReportingActiveUsersToWarehouse
-    load-internal-database = edx.analytics.tasks.warehouse.load_internal_reporting_database:ImportMysqlDatabaseFromS3ToVerticaSchemaTask
```
Comment (HassanJaveed84) [inline setup.cfg:72]: We would still need to keep an entry point to `load_internal_reporting_database` and retain that file with code necessary for sqooping out mysql database to S3 and loading into snowflake.

File: setup.cfg [hunk 4]
```diff
@@ -56,36 +55,8 @@ edx.analytics.tasks =
...
-
-    # financial:
-    cybersource = edx.analytics.tasks.warehouse.financial.cybersource:DailyPullFromCybersourceTask
-    financial_reports  = edx.analytics.tasks.warehouse.financial.finance_reports:BuildFinancialReportsTask
-    orders = edx.analytics.tasks.warehouse.financial.orders_import:OrderTableTask
-    payment_reconcile = edx.analytics.tasks.warehouse.financial.reconcile:ReconcileOrdersAndTransactionsTask
-    paypal = edx.analytics.tasks.warehouse.financial.paypal:PaypalTransactionsByDayTask
-    paypal-report = edx.analytics.tasks.warehouse.financial.paypal_ftpreport:LoadPayPalCaseReportToVertica
-    affiliate_window = edx.analytics.tasks.warehouse.financial.fees:LoadFeesToWarehouse
+    load-course-catalog = edx.analytics.tasks.warehouse.load_internal_reporting_course_catalog:PullDiscoveryCoursesAPIData
```
Comment (HassanJaveed84) [inline setup.cfg]: We probably don't need to keep this.

Comment (bmedx) [reply setup.cfg]: I just re-added that because @brianhw said we did, so I'm tempted to just leave it for now. :)

File: setup.cfg (hunk 3 above)
Comment (bmedx) [reply setup.cfg:72]: Added that back in, also removed a bunch of lingering BigQuery references.

File: setup.cfg [hunk 5]
```diff
@@ -56,36 +54,9 @@ edx.analytics.tasks =
...
+    load-internal-database = edx.analytics.tasks.warehouse.load_internal_reporting_database:LoadMysqlTableFromS3ToSnowflakeTask
```
Comment (bmedx) [inline setup.cfg:58]: @HassanJaveed84 I've changed the entry point to this, but am not familiar with how entry points are being used here. Does this seem like a reasonable change?

File: setup.cfg (hunk 4 above)
Comment (brianhw) [reply setup.cfg]: Yeah, this is my fault.  We link to tasks that are defined in this file, and I was concerned that the tasks would not be found if not added to Stevedore.   But that's only needed for entry points, and we no longer use anything in this file as an entry point anymore.   It becomes like any of the utility files.   So I think Hassan is right.

**Review by brianhw — APPROVED** [review]
LGTM.   👍

File: setup.cfg (hunk 5 above)
Comment (HassanJaveed84) [reply setup.cfg:58]: This should suffice.

**Pull request merged**